common/util.go

```diff
@@ -39,6 +39,15 @@ var (
 	ErrFormatProto = fmt.Errorf("unknown VideoProfile format for protobufs")
 	ErrFormatMime  = fmt.Errorf("unknown VideoProfile format for mime type")
 	ErrFormatExt   = fmt.Errorf("unknown VideoProfile format for extension")
+
+	ext2mime = map[string]string{
+		".ts":   "video/mp2t",
+		".mp4":  "video/mp4",
+		".mpg":  "video/mpeg",
+		".ogv":  "video/ogg",
+		".mov":  "video/quicktime",
+		".m3u8": "application/x-mpegurl",
```

Comment:
Can we limit ourselves to formats that are actually defined in LPMS `ffmpeg.Format` ? Meaning, just ts and mp4 for now.

Reply:
fixed